Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -48,33 +48,22 @@ set_option linter.deprecated false
 section
 variable {f : Bool → Bool → Bool}
 
-lemma bitwise'_bit' {f : Bool → Bool → Bool} (a : Bool) (m : Nat) (b : Bool) (n : Nat)
-    (ham : m = 0 → a = true) (hbn : n = 0 → b = true) :
-    bitwise' f (bit a m) (bit b n) = bit (f a b) (bitwise' f m n) := by
```

Comment:
Is there a version of this theorem for `bitwise`?

Reply:
There is `bitwise_bit`, which is a similar theorem, but with `f false false` as the hypothesis. This theorem was only added to support the now-removed proof of equality between the `bitwise`s.

I could bring it back with the alternate hypotheses, if you think that's usefull?

Reply:
I think it is useful, and we tend to have a rule that mathlib shouldn't lose theorems.

Reply:
I was hoping it wouldn't be too difficult to prove.

Reply:
It was not too bad, we now have a proof of `bitwise_bit'` as well